crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -678,53 +682,106 @@ impl TryFrom<TransactionEffects> for SuiTransactionBlockEffects {
     }
 }
 
+fn owned_objref_string(obj: &OwnedObjectRef) -> String {
+    format!(
+        " ┌──\n │ ID: {} \n │ Owner: {} \n │ Version: {:?} \n │ Digest: {}\n └──",
+        obj.reference.object_id, obj.owner, obj.reference.version, obj.reference.digest
```

Comment:
We also don't want the redundant `SequenceNumber(...)`

```suggestion
        " ┌──\n │ ID: {} \n │ Owner: {} \n │ Version: {} \n │ Digest: {}\n └──",
        obj.reference.object_id, obj.owner, obj.reference.version.value(), obj.reference.digest
```